scripts/exgdas_global_marine_analysis_run.sh

```diff
@@ -30,6 +30,30 @@ pwd=$(pwd)
 #  Utilities
 export NLN=${NLN:-"/bin/ln -sf"}
 
+# function that converts a soca incr to something MOM6 is happy with
+function socaincr2mom6 {
+  incr=$1
+  bkg=$2
+  grid=$3
+  incr_out=$4
+
+  scratch=scratch_socaincr2mom6
+  mkdir -p $scratch
+  cd $scratch
+
+  echo "at socaincr2mom6" $bkg $grid
+
+  cp $incr inc.nc                   # TODO: use accumulated incremnet, not outerloop intermediates
```

Comment:
remove comment once you point to the total increment.

Reply:
updated

Reply:
doesn't look like it, I still see `# TODO: use accumulated incremnet, not outerloop intermediates` ... which presumably was done